src/autopas/pairwiseFunctors/FlopCounterFunctor.h

```diff
@@ -30,6 +30,8 @@ class FlopCounterFunctor : public Functor<Particle, FlopCounterFunctor<Particle>
 
   bool allowsNonNewton3() override { return true; }
 
+  bool allowsMixedNewton3() override { return false; }
```

Comment:
done in [4a5aece](https://github.com/AutoPas/AutoPas/pull/745/commits/4a5aece0f8222be36761514e6c229ecbf982c33f), since allowsMixedNewton3() is removed again.